.travis.yml

```diff
@@ -1,20 +1,17 @@
 language: go
 
 go:
-  - 1.6
-  - 1.7
-  - 1.8
-  - 1.9
+  - 1.11
 
 install:
-- curl -L https://github.com/golang/dep/releases/download/v0.3.2/dep-linux-amd64 -o dep
-- chmod u+x dep && ./dep ensure
-- go get -u github.com/alecthomas/gometalinter
-- gometalinter -u -i -f
+- curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $GOPATH/bin v1.10.2
 
 script:
 - go test -race -coverprofile=coverage.txt -covermode=atomic
-- gometalinter --vendor --config=./gometalinter.json ./...
+- golangci-lint run --fast
```

Comment:
Could you add a .golangci.yml with following content:

```yaml
linters-settings:
  lll:
    line-length: 150

linters:
  enable-all: true

issues:
  max-same: 0 # Disable maximum count of issues with the same text.
  exclude:
  - G104 # Duplicate of errcheck

  exclude-use-default: false
```

And remove the `--fast` option ?

By default, golangci does not enable golint (which was enabled in our `gometalinter.json` file)